src/main/java/org/icgc_argo/workflow/relay/config/ProfileManager.java

```diff
@@ -0,0 +1,17 @@
+package org.icgc_argo.workflow.relay.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Configuration
+public class ProfileManager {
+  @Value("${spring.profiles.active:}")
```

Comment:
The accepted way of getting what profiles you want is to use dependency injection for the spring `Environment` and then use the public method provided there. 

```java
@Configuration
public class MyClass {
  
  List<String> profiles;
  
  public MyClass(@Autowired Environment environment) {
    profiles = asList(environment.getActiveProfiles());
  }
}
```